generator/generator.yml

```diff
@@ -612,3 +612,42 @@ modules:
         lookup: lgpPduRcpEntrySysAssignLabel
       - source_indexes: [lgpPduAuxSensorIndex]
         lookup: lgpPduAuxSensorSysAssignLabel
+# Mikrotik Router
+#
+# http://download2.mikrotik.com/Mikrotik.mib
+  mikrotik:
+    walk:
+      - interfaces
+      - ifMIB
+      - laIndex
+      - sysUpTime
+      - sysDescr
+      - host
+      - mikrotik
+    lookups:
+      - source_indexes: [ifIndex]
+        lookup: ifAlias
+      - source_indexes: [ifIndex]
+        lookup: ifDescr
+      - source_indexes: [ifIndex]
+        lookup: 1.3.6.1.2.1.31.1.1.1.1
+      - source_indexes: [mtxrInterfaceStatsIndex]
+        lookup: ifAlias
+      - source_indexes: [mtxrInterfaceStatsIndex]
+        lookup: ifDescr
+      - source_indexes: [mtxrInterfaceStatsIndex]
+        lookup: 1.3.6.1.2.1.31.1.1.1.1
```

Comment:
This doesn't seem right, what are you trying to do?

Reply:
True, replaced by ifNames.